docs/source/concepts/bento.rst

```diff
@@ -204,6 +204,66 @@ Bentos between teams or moving between different deployment stages. For example:
     `the list <https://www.pyfilesystem.org/page/index-of-filesystems/>`_ provided by the
     pyfilesystem library.
 
+Test Bentos
+^^^^^^^^^^^
+
+After you build a Bento, it's essential to test it locally before containerizing it or pushing it to BentoCloud
+for production deployment. Local testing ensures that the Bento behaves as expected and helps identify any potential
+issues. Here are two methods to test a Bento locally.
+
+Use the BentoML CLI
+"""""""""""""""""""
+
+You can easily serve a Bento using the BentoML CLI. Replace ``BENTO_TAG`` with your specific Bento tag (for example, ``iris_classifier:latest``) in the following command.
+
+.. code:: bash
+
+    bentoml serve BENTO_TAG
+
+Use the bentoml.Server API
+""""""""""""""""""""""""""
+
+For those working within scripting environments or running Python-based tests where using the CLI might be
+difficult, the ``bentoml.Server`` API offers a more programmatic way to serve and interact with your Bento.
+It gives you detailed control over the server lifecycle, especially useful for debugging and iterative testing.
```

Comment:
I think they idea here is if they are writing pytest or unit tests, it is best to use `bentoml.Server`?

If that is the case then I suggest the following:

```prolog
For writing unit/integration tests with Bento, it is recommended to use `bentoml.Server` API as it offers a programmatic way to manage the lifecycle of a BentoServer.
```

Reply:
I think this wording is fine.